### PR TITLE
Add Go to IPNS modal

### PR DESCRIPTION
**Pull request by 0x6431346e**

### CODE REVIEW

File: app/windows/GoToIPNS/renderer.jsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+          <Content>
+            <Input
+              label="IPNS"
+              type="text"
+              placeholder="the name to be resolved"
+            />
+          </Content>
+          <Toolbar ptType="footer">
+            <Actionbar>
+              <Button text="Open" ptStyle="primary" type="submit" pullRight />
```
Comment (koalalorenzo) [inline app/windows/GoToIPNS/renderer.jsx]: Disable the Open button if the IPNS is still resolving

Comment (koalalorenzo) [reply app/windows/GoToIPNS/renderer.jsx]: Maybe also a loading-spinning icon?

File: app/api.test.js [hunk 2]
```diff
@@ -40,6 +40,35 @@ describe('api.js', function () {
...
+      api.setClientInstance(null)
+      // act
+      return api.resolveName()
+        .catch(err => {
+          // assert
+          expect(err).toBe(ERROR_IPFS_UNAVAILABLE)
+        })
+    })
+
+    it('should resolve the given name', function () {
```
Comment (koalalorenzo) [inline app/api.test.js]: Should we add a case when the IPNS is not available?

File: app/menu.js [hunk 3]
```diff
@@ -6,15 +6,21 @@ import electron, { BrowserWindow, Menu, app } from 'electron'
...
   submenu: [{
     label: 'Import from hash',
     accelerator: 'CmdOrCtrl+D',
     click () {
-      const importWindow = ImportWindow.create(app)
-      importWindow.show()
+      ImportWindow.create(app)
+    }
+  }, {
+    label: 'Go to IPNS',
```
Comment (koalalorenzo) [inline app/menu.js]: Rename to "Resolve IPNS" as we are not opening the file, but opening a "DetailWindow". I have created a new issue for that: https://dev.siderus.io/issues/49

File: app/api.test.js (hunk 2 above)
Comment (0x6431346e) [reply app/api.test.js]: We don't catch the error though, so we wouldn't be testing the implementation. Should we catch `.name.resolve` and give an custom error?

Comment (koalalorenzo) [reply app/api.test.js]: It is fine for now

**Pull request merged**